### ISSUE TITLE
WP Rocket: WP_CACHE constant

### ISSUE THREAD

**Issue by brettsmason** [eighteen73/nebula#5]
Look at a way to handle WP Rockets in built way of adding WP_CACHE to web/wp-config.php.

The easiest way is with a filter:

```
add_filter( 'rocket_set_wp_cache_constant', '__return_false' );
```

To decide where this lives - in Nebula or Orbit.

**Comment by edjeavons** [comment]
My gut says it should go in Orbit because that's where we can be a little more opinionated about other plugins that may be installed on the website. What do you think, @brettsmason?

**Comment by brettsmason** [comment]
Yes agreed, it should live in Orbit. We want to avoid anything being defined within wp-config.php
